README.md

```diff
@@ -1 +1,59 @@
-# terraform-aws-named-subnets
\ No newline at end of file
+# terraform-aws-named-subnets
+
+Terraform module for `named subnets` provisioning.
+
+
+## Usage
+
+```terraform
+module "named_subnets" {
+  source                    = "git::https://github.com/cloudposse/terraform-aws-named-subnets.git?ref=master"
+  namespace                 = "${var.namespace}"
+  stage                     = "${var.stage}"
+  public_names              = ["${var.public_names}"]
```

Comment:
looks like we're go round in circles

<img width="792" alt="screenshot 2017-10-23 20 59 51" src="https://user-images.githubusercontent.com/26582191/31904944-403cf930-b835-11e7-8f39-9ba4c5a7a1b7.png">

Reply:
A bit.
I think, module should has own purpose.
#1 module is too common. For me it is almost no difference between plain `aws_subnet` with count and #1 
When this PR #2 creates a ready subnets topology.

It can be "merged" with [`terraform-aws-dynamic-subnets`](https://github.com/cloudposse/terraform-aws-dynamic-subnets) later.
And we would have a module, which allows to create any amount of subnets within any amount of AZs.
This "merge" will make module complex, but provide simple "user interface".